.github/BOTMETA.yml

```diff
@@ -126,7 +126,7 @@ files:
     maintainers: giner
   $filters/from_csv.py:
     maintainers: Ajpantuso
-  $filters/groupby:
+  $filters/groupby.py:
     maintainers: felixfontein
   $filters/hashids:
```

Comment:
Should `hashids` also have an extension? I'm not familiar with this file, just basing on the change to `groupby`.

Reply:
`hashids` is correct, since that matches `hashids.py`, `hashids_encode.yml` and `hashids_decode.yml`. But having separate entries is better IMO.